### PR TITLE
Chore/kill it with fire

### PR DESCRIPTION
**Pull request by JohnAllenTech**
# ✨ Codu Pull Request 💻

Codu Logo

## Pull Request details

- Prisma clean up

## Any Breaking changes

- None

## Associated Screenshots

Lord of the rings Frodo fire meme

## Summary by CodeRabbit

- **Refactor**
	- Streamlined the build process by optimizing configurations and updating build scripts.
	- Transitioned database operations to use Drizzle instead of Prisma, affecting how tables are set up and data queries are performed.
- **Chores**
	- Updated `.gitignore` and `Dockerfile` to align with new technology stack changes.
- **Documentation**
	- Updated README.md to reflect the change in database technology from Prisma to Drizzle.

### CODE REVIEW

File: sst.config.ts [hunk 1]
```diff
@@ -32,7 +32,6 @@ export default {
         );
       }
       const site = new NextjsSite(stack, "site", {
-        // @TODO: Fix Prisma bundle issue
```
Comment (JohnAllenTech) [inline sst.config.ts:35]: This should love the bundle issue?

File: server/api/trpc.ts [hunk 2]
```diff
@@ -46,8 +45,6 @@ export const createInnerTRPCContext = async (opts: CreateContextOptions) => {
     session,
     headers: opts.headers,
     db,
-    // Phase out Prisma
```
Comment (JohnAllenTech) [inline server/api/trpc.ts:49]: Mission accomplished

File: Dockerfile [hunk 3]
```diff
@@ -9,7 +9,6 @@ WORKDIR /app
 
 # Install dependencies based on the preferred package manager
 COPY package.json package-lock.json* ./
-COPY prisma ./prisma
```
Comment (JohnAllenTech) [inline Dockerfile:12]: No longer need this as dont need a prisma client anymore

File: drizzle/seed.ts [hunk 4]
```diff
@@ -339,7 +339,6 @@ const main = async () => {
     "C",
     "React",
     "Drizzle",
-    "Prisma",
```
Comment (JohnAllenTech) [inline drizzle/seed.ts:342]: This was petty but I stand by it

**Review by NiallJoeMaher — APPROVED** [review]
LFG! 🌮 And bonus 🌮  for memes

**Pull request merged**